modules/learn/partials/maintain-durability-warning.adoc

```diff
@@ -0,0 +1,10 @@
+[WARNING]
+.Potential Data Loss
+====
+Enabling `durabilityImpossibleFallback` degrades the promise that durable writes offer: that Couchbase Server has persisted the data in a way that should survive node failure.
```

Comment:
I would replace "promise" with "guarantee" as that's the nomenclature we have otherwise used.